src/script/bitcoinconsensus.h

```diff
@@ -72,6 +72,14 @@ EXPORT_SYMBOL int bitcoinconsensus_verify_script_with_amount(const unsigned char
                                     const unsigned char *txTo        , unsigned int txToLen,
                                     unsigned int nIn, unsigned int flags, bitcoinconsensus_error* err);
 
+/// Returns 1 if the input nIn of the serialized transaction pointed to by
+/// txTo correctly spends the scriptPubKey pointed to by scriptPubKey under
+/// the current standardness constraints.
```

Comment:
In commit "[policy] Extend bitcoinconsensus library with standard script flags" (a7a0c4c7612efbf95849cd70c28497d27b3386c2)

Maybe extend comment to say something about limitations of this function, e.g. that it only reflects default policy in the current release, actual policy can vary between nodes, software versions